index.html

```diff
@@ -0,0 +1,51 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>👨🏻‍💻 Mauricio J. Monta | Front-end Dev</title>
+    <link rel="stylesheet" href="style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+<link href="https://fonts.googleapis.com/css2?family=Poppins:ital,wght@0,100;0,200;0,300;0,400;0,500;0,600;0,700;0,800;0,900;1,100;1,200;1,300;1,400;1,500;1,600;1,700;1,800;1,900&display=swap" rel="stylesheet">
+</head>
+<body>
+    <!--toolbar-->
+    <header id="navbar">
+        <div id="toolbar" class="toolbar">
+          <a class="logolink" href="#"><h2 id="toolbartitle">My Logo</h2></a>
+        </div>
+        <nav>
+          <button type="button" id="hamBtn">
+            <img src="imgs/hambtn.svg" height="12" width="18" alt="hamburger" />
+          </button>
+        </nav>
+      </header>
+      <header class="links" id="links">
+        <button type="button" id="hamBtnClose">X</button>
+        <ul class="nav__links" id="mobile-menu">
+          <li><a href="#portfoliosection">Portfolio</a></li>
+          <li><a href="#aboutmesection">About</a></li>
+          <li><a href="#contact">Contact</a></li>
+        </ul>
+      </header>
+        <!--toolbar-->
+        <!--headline-->
+        <div class="background">
+        <section class="headline">
+            <h2>I'm Mauricio Glad to see you!</h2>
+            <p id="hl-paragraph">I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+            <button type="button" class="hl-button">Let's Connect</button>
+                <div class="socialmedia-icons">
+                    <a href="https://www.twitter.com"><img src="imgs/twitter.svg" alt="Twitter Icon"></a>
+                    <a href="https://www.github.com"><img src="imgs/github.svg" alt="Github Icon"></a>
+                    <a href="https://www.linkedin.com"><img src="imgs/linkedin.svg" alt="LinkedIn Icon"></a>
+                </div>
```

Comment:
- [ ] As this is a list of elements, I suggest you use an unordered list tag `<ul>`, with each element being a list item `<li>`.